src/modm/driver/can/mcp2515.hpp

```diff
@@ -186,6 +198,21 @@ namespace modm
 			BIT_MODIFY = 0x05
 		};
 
+		static void
+		mcp2515interrupt();
+
+		static bool
+		mcp2515readMessage(can::Message& message);
+
+		bool
+		mcp2515isReadyToSend(uint8_t status);
+
+		modm::ResumableResult<bool>
+		mcp2515isReadyToSend();
+
+		modm::ResumableResult<bool>
+		mcp2515sendMessage(const can::Message& message, const uint8_t status = 0xff);
```

Comment:
This are like "internal" functions? Please make them private and more descriptive naming would be good too.
(also: _mcp2515**R**eadMessage()_)

Reply:
these were protected in the original code, so i left the classifier as it was before. i will change it to private

Reply:
Ahh nevermind, I got confused by the github diff rendering. Protected should be fine.